client/client.go

```diff
@@ -133,6 +134,11 @@ type RPCClient interface {
 	// SetExternalTimestamp sets external timestamp
 	SetExternalTimestamp(ctx context.Context, timestamp uint64) error
 
+	// ManuallyChangePeer manually changes the peer of a region.
+	ManuallyChangePeer(ctx context.Context, regionID uint64, changeType eraftpb.ConfChangeType, peer *metapb.Peer) error
```

Comment:
Is it necessary? Why not let TiKV handle it? If we are going to use RPC to solve the snapshot problem, we still can not ensure that the operator is executed successfully.

Reply:
It's used to make the operation on `add peer` / `remove peer` managed by PD uniformly.

If let TiKV handle it directly, the scheduling on `add peer` or `remove peer` will not be perceived by PD, and the check of the scheduler cannot be validated by the `store limit` and other checking mechanisms.